scripts/install-helpers/README.md

```diff
@@ -0,0 +1,145 @@
+# Introduction
+
+These are helper scripts to setup Trustee operator on OpenShift cluster
+
+## Prerequisites
+
+- `oc`, `jq` and `openssl` CLI
+
+## Install Trustee operator GA release
+
+- Update `startingCSV` key in the `subs-ga.yaml` file to use the GA release you need.
+
+- Kickstart the installation by running the following:
+
+```sh
+./install.sh
+```
+
+This will install the Trustee operator and create `KbsConfig` CRD with an opinionated
+configuration. Specifically it will create a key-pair for authentication to KBS,
+create configMaps for KBS, RVPS, resource policy and a sample K8s secret `kbsres1`.
+All the resources will be created under the `trustee-operator-system` namespace.
+
+## Install Trustee operator pre-GA release
+
+- Update trustee_catalog.yaml to point to the pre-GA catalog
+  
+- The pre-GA build images are in an authenticated registry, so you'll need to
+  set the `PULL_SECRET_JSON` variable with the registry credentials. Following is an example:
+
+  ```sh
+  export PULL_SECRET_JSON='{"brew.registry.redhat.io": {"auth": "abcd1234"}, "registry.redhat.io": {"auth": "abcd1234"}}'
+  ```
+
+- Kickstart the installation by running the following:
+
+  ```sh
+  ./install.sh -m -s -b
+  ```
+
+  This will deploy the pre-GA release of Trustee operator.
+
+## Un-installation
+
+Run the following command to uninstall
+
+For TDX hosts:
+
+```sh
+./install.sh -u
+```
+
+## Optional configurations
+
+### Permissive resource policy
+
+For testing with sample attester, you will have to do this:
+
+Get the KBS deployment pod name
+
+```sh
+POD_NAME=$(oc get pods -l app=kbs -o jsonpath='{.items[0].metadata.name}' -n trustee-operator-system)
+```
+
+Allow all access to resources
+
+```sh
+oc exec -n trustee-operator-system -it "$POD_NAME" -c kbs  -- sed -i 's/false/true/g' /opa/confidential-containers/kbs/policy.rego
```

Comment:
I don't think you can edit the file, it is a r/o directory because of the config map. Edit the config map instead? Also, the default policy path is `/opt/confidential-containers/opa/policy.rego`

Reply:
Right. This is left over from old one..